Example/ReactiveDataDisplayManager/Table/CustomizableComponentsTableViewController/CustomizableComponentsTableViewController.swift

```diff
@@ -0,0 +1,104 @@
+
+    private lazy var recievedTimeMessageStyle: LabelView.TextStyle = (.gray, .systemFont(ofSize: 12, weight: .light))
+    private lazy var recievedTimeMessageLayout: LabelView.TextLayout = (.right, .byWordWrapping, 0)
+    private lazy var recievedTimeMessageModel: LabelView.Model = .init(text: "17:32",
+                                                                       style: recievedTimeMessageStyle,
+                                                                       layout: recievedTimeMessageLayout,
+                                                                       edgeInsets: UIEdgeInsets(top: 12, left: 0, bottom: 12, right: 16))
+    private lazy var recievedMessageTimeGenerator = BaseCellGenerator<TableLabelCell>(with: recievedTimeMessageModel, registerType: .class)
+
+    private lazy var dateStyle: LabelView.TextStyle = (.black, .systemFont(ofSize: 12, weight: .light))
+    private lazy var dateLayout: LabelView.TextLayout = (.center, .byWordWrapping, 0)
+    private lazy var dateModel: LabelView.Model = .init(text: "24 мая 2023",
+                                                        style: dateStyle,
+                                                        layout: dateLayout,
+                                                        edgeInsets: UIEdgeInsets(top: 0, left: 0, bottom: 12, right: 0))
+    private lazy var dateGenerator = BaseCellGenerator<TableLabelCell>(with: dateModel, registerType: .class)
+
+    private lazy var sentMessageStyle: LabelView.TextStyle = (.black, .systemFont(ofSize: 16, weight: .regular))
+    private lazy var sentMessageLayout: LabelView.TextLayout = (.left, .byWordWrapping, 0)
+    private lazy var sentMessageModel: LabelView.Model = .init(text: "Lorem ipsum dolor sit amet",
+                                                               style: sentMessageStyle,
+                                                               layout: sentMessageLayout,
+                                                               edgeInsets: UIEdgeInsets(top: 12, left: 16, bottom: 12, right: 0))
+    private lazy var sentMessageGenerator = BaseCellGenerator<TableLabelCell>(with: sentMessageModel, registerType: .class)
+
+    private lazy var recievedMessageStyle: LabelView.TextStyle = (.black, .systemFont(ofSize: 16, weight: .regular))
+    private lazy var recievedMessageLayout: LabelView.TextLayout = (.right, .byWordWrapping, 0)
+    private lazy var recievedMessageModel: LabelView.Model = .init(text: "Lorem ipsum dolor sit amet",
+                                                                   style: recievedMessageStyle,
+                                                                   layout: recievedMessageLayout,
+                                                                   edgeInsets:  UIEdgeInsets(top: 12, left: 0, bottom: 12, right: 16))
+    private lazy var recievedMessageGenerator = BaseCellGenerator<TableLabelCell>(with: recievedMessageModel, registerType: .class)
```

Comment:
```suggestion
    private lazy var recievedMessageGenerator =TableLabelCell.rddm.baseGenerator(with: recievedMessageModel, registerType: .class)
```
Можно еще так)
Этот способ предпочтительней.